README.md

```diff
@@ -11,71 +11,73 @@
 
 ## Goals
 
-Syncthing is a **continuous file synchronization program**. It synchronizes
-files between two or more computers. We strive to fulfill the goals below.
+Syncthing is a **continuous file synchronization program**.  
+It synchronizes files between two (or more) computers. 
```

Comment:
What's the benefit of parentheses here?

Also, please minimize the re-wrapping at arbitrary locations.  English sentences can be concatenated into a paragraph.  No need to start each one on a new line.  Plus you're leaving behind trailing spaces on every line, please don't do that.